src/test/groovy/graphql/execution/ValuesResolverTest.groovy

```diff
@@ -603,6 +603,40 @@ class ValuesResolverTest extends Specification {
 
     }
 
+    def "getArgumentValues: invalid oneOf input validation applied with list of input type"() {
+        given: "schema defining input object"
+        def inputObjectType = newInputObject()
+                .name("oneOfInputObject")
+                .withAppliedDirective(Directives.OneOfDirective.toAppliedDirective())
+                .field(newInputObjectField()
+                        .name("a")
+                        .type(GraphQLString)
+                        .build())
+                .field(newInputObjectField()
+                        .name("b")
+                        .type(GraphQLInt)
+                        .build())
+                .build()
+
+        def inputValue = buildObjectLiteral([
+                oneOfField: [
+                        a: StringValue.of("abc"),
+                        b: IntValue.of(123)
+                ]
+        ])
```

Comment:
While I agree that we haven't handled the case where there is a list of input objects, I think this test case has to be adjusted. The type on line 632 doesn't match with the type presented in the literal. 

Later on line 632 we pass in the `type` as a `list(inputObjectType)`, but `inputValue` is not an `inputObjectType`, it's got one more layer of wrapping.

Found this by setting breakpoints and finding that the `Object inputValue` passed into `validateOneOfInputTypes` was coming up as null rather than the expected list of maps.

Can fix by removing extra wrap layer, or alternatively could change the list specified in the argument on line 632
```
def inputValue = buildObjectLiteral([
    a: StringValue.of("abc"),
    b: IntValue.of(123)])
```